support/Pipelines/Bbh/InitialData.yaml

```diff
@@ -115,6 +115,7 @@ DomainCreator:
       InitialTime: 0.
       ExpansionMap: None
       RotationMap: None
+      TranslationMap: None
```

Comment:
Oh actually wait. All the input file changes that have `TranslationMap` should go in the first commit, otherwise it won't compile correctly on its own. The only input file changes in the second commit should be the translation control system. That was my bad for saying the wrong thing above